jaiarobotics/jaiabot
# feature/add-full-screen-toggle

## Summary
A new update to the tablets prevents the home screen icon from displaying in full screen. For mobile and tablet devices, there will now be a toggle in the settings panel to allow operators to enter a full size view.

## Testing
* Toggle does not appear on laptop
* Toggle appears on Samsung S6 tablet
* Toggle works as expected on tablet
